csfieldguide/package.json

```diff
@@ -18,8 +18,11 @@
     "csfg-interactive-caesar-cipher": "file:./static/interactives/caesar-cipher/",
     "csfg-interactive-cmy-mixer": "file:./static/interactives/cmy-mixer/",
     "csfg-interactive-colour-matcher": "file:./static/interactives/colour-matcher/",
+    "csfg-interactive-dictionary-compression": "file:./static/interactives/dictionary-compression",
```

Comment:
These don't need to be added if the package.json files don't have any dependencies